### PR TITLE
Fix golint failures on cmd/kubeadm/app

### PR DESCRIPTION
**Pull request by SataQiu**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
fix golint failures on cmd/kubeadm/app

**Which issue(s) this PR fixes**:
Ref #68026

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

**Review by yagonobre — COMMENTED** [review]
Hey @SataQiu, thanks for you pr.
Can you change your pr description from `Fixes #68026` to `Ref #68026`?

/priority important-longterm

**Comment by SataQiu** [comment]
@yagonobre Thank you! I have modified the description.

File: cmd/kubeadm/app/kubeadm.go [hunk 1]
```diff
@@ -21,6 +21,8 @@ import (
 	"os"
 
 	"github.com/spf13/pflag"
+
+	// ensure libs have a chance to globally register their flags
```
Comment (neolit123) [inline cmd/kubeadm/app/kubeadm.go]: please change to:
> // allow klog to register its global flags

**Review by neolit123 — COMMENTED** [review]
/approve
/kind cleanup
/remove-priority important-longterm
/priority backlog
/hold

@kubernetes/sig-cluster-lifecycle-pr-reviews 
objections?

**Review by yagonobre — COMMENTED** [review]
LGTM with the new comment

**Comment by neolit123** [comment]
/assign @fejta 

this is just a small lint fix. needs approve for `/hack`.
thanks.

**Pull request merged**